<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_Slamware</class>
 <widget class="QMainWindow" name="C_Slamware">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>502</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>419</width>
    <height>502</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>419</width>
    <height>502</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Slamware Interface</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="gv_LdsScan">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ZoomDefault">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>450</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>O</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ZoomOut">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>450</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ZoomIn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>450</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ShowComPort">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>440</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show COM port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Connect">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>470</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_CenterLdsScan">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_MoveUpLdsScan">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>v</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_MoveDownLdsScan">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_MoveLeftLdsScan">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>450</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_MoveRightLdsScan">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_RefreshRate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>lbl_RefreshRate</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
